setup.py

```diff
@@ -140,5 +137,5 @@ def run_tests(self):
         "pytest",
     ],
     url="https://github.com/CellProfiler/centrosome",
-    version="1.0.9",
+    version="1.1.0-dev",
```

Comment:
We should go ahead and use `1.1.0` (or even `2.0.0`).

Reply:
@0x00b1 at scikit-image we prefer to use -dev except for release branches, so if someone says "v0.13.1" we know they are not on some weird between-releases commit. But, no problem, bumped to 2.0.0. =)